Blocks/URL/url.h

```diff
@@ -445,6 +447,64 @@ inline URLPathArgs::CountMask& operator<<=(URLPathArgs::CountMask& lhs, const si
   return lhs;
 }
 
+
+constexpr char URLPathSeparator = '/';
+
+inline char GetURLPathSeparator() {
```

Comment:
Why do we need a getter instead of directly using `URLPathSeparator` as constant?